src/BullOak.Repositories/Session/BaseRepoSession.cs

```diff
@@ -22,13 +21,22 @@ public abstract class BaseRepoSession<TState> : IManageSessionOf<TState>
         private TState currentState;
         public TState GetCurrentState() => currentState;
 
+        protected readonly IValidateState<TState> stateValidator = new AlwaysPassValidator<TState>();
```

Comment:
Instead of field initializer here could use redirect to the new ctor in L40:

```csharp
protected BaseRepoSession(IHoldAllConfiguration configuration, IDisposable disposableHandle)
 : this(new AlwaysPassValidator<TState>(), configuration, disposableHandle)
```